modules/5_install/install.tf

```diff
@@ -207,6 +207,7 @@ resource "null_resource" "config" {
   provisioner "remote-exec" {
     inline = [
       "sed -i \"/^helper:.*/a \\ \\ networkifacename: $(ip r | grep \"${var.cidr} dev\" | awk '{print $3}')\" ocp4-helpernode/helpernode_vars.yaml",
+      "sed -i 's/when: ansible_distribution_major_version == \"8\"/when: ansible_distribution_major_version >= \"8\"/g' ocp4-helpernode/tasks/set_facts_.yaml",
```

Comment:
helpernode owner is not allowing this change? should we work on the fork for our team? Let me know your thoughts.

Reply:
I believe this change is already merged in the helpernode repo https://github.com/redhat-cop/ocp4-helpernode/pull/291

Reply:
Perhaps not promoted to the main branch.

Reply:
Yes

Reply:
yes, it is not in main branch yet, it still works even if it merged to main branch.